fineract-provider/src/main/java/org/apache/fineract/batch/command/internal/AdjustLoanTransactionByExternalIdCommandStrategy.java

```diff
@@ -45,7 +45,7 @@
  */
 @Component
 @RequiredArgsConstructor
-public class AdjustTransactionByExternalIdCommandStrategy implements CommandStrategy {
+public class AdjustLoanTransactionByExternalIdCommandStrategy implements CommandStrategy {
 
     /**
```

Comment:
Arent these changes irrelevant to [FINERACT-1921](https://issues.apache.org/jira/browse/FINERACT-1921)? Please extract them into a separate PR if so!